pipelines/full-test-nas.yml

```diff
@@ -1,24 +1,57 @@
 trigger: none
-pr: none
+
 schedules:
 - cron: 0 16 * * *
   branches:
     include: [ master ]
 
-jobs:
-- job: linux
-  pool: nni-it
-  timeoutInMinutes: 60
+resources:
+  repositories:
+    - repository: azpfilter
+      type: github
+      name: microsoft/azure-pipeline-filter
+      endpoint: github-filter-connection
+
+variables:
+  filter.modified.globs: 'examples/nas/**,nni/algorithms/nas/**,nni/nas/**,nni/retiarii/**'
+  filter.prbody.heading: '#### Test Options'
+  filter.prbody.optionIndex: 0
```

Comment:
All 0?

Reply:
updated template

### Description ###

#### Test Options ####
  - [ ] fast test
  - [ ] full test - HPO
  - [ ] full test - NAS
  - [ ] full test - compression

### Checklist ###
  - [ ] test case
  - [ ] doc

### How to test ###